src/modes/overview/vegaSpec.js

```diff
@@ -121,10 +126,49 @@ export function createSpec(sensor, selections, dateRange, valuePatch) {
     transform: [
       ...(sensor.hasStdErr ? (isPercentage ? stdErrTransformPercent : stdErrTransform) : []),
       {
-        calculate: 'datum.value == null ? null : datum.value / 100',
         as: 'pValue',
+        calculate: 'datum.value == null ? null : datum.value / 100',
+      },
+      {
+        as: 'clipped',
+        calculate: !clipping + '|| datum.' + yField + ' == null ? false : datum.' + yField + ' > ' + yMaxScaled,
```

Comment:
consider using backticks (``) for better readability of string interpolation

Reply:
Yes, it seemed easier while I was experimenting with many variations to just use concatenation, but actually, backticks everywhere would have been easier.